src/substrate/types.ts

```diff
@@ -188,6 +191,13 @@ interface IEvent {
   kind: EventKind;
 }
 
+export interface IBalanceTransfer extends IEvent {
+  kind: EventKind.BalanceTransfer;
+  transactor: AccountId;
```

Comment:
Is `transactor` the best word? I know on Ethereum the term is `sender`.

Reply:
That's what they use in the substrate source 🤷 open to changing it though.